batchflow/models/torch/base.py

```diff
@@ -124,16 +124,18 @@ class TorchModel:
         - ``{'optimizer': 'Adam'}``
         - ``{'optimizer': ('SparseAdam', {'lr': 0.01})}``
         - ``{'optimizer': {'name': 'Adagrad', 'initial_accumulator_value': 0.01}``
-        - ``{'optimizer': {'name': MyCustomOptimizer, momentum=0.95}}``
+        - ``{'optimizer': {'name': MyCustomOptimizer, momentum: 0.95}}``
 
     decay : str, tuple, dict
         Learning rate decay algorithm, might be defined in multiple formats.
-        All decays require to have ``n_iters`` as a key in a configuration
+        All decays require to have 'step' as a key in a configuration
```

Comment:
As far as I know, that is not the correct explanation of what `step` is.
Anyways, I still think that `step` is not correct key-word for `apply decay every that iterations` parameter, and I still think that `frequency` is a better one

Reply:
done